app/src/main/java/com/devindi/wallpaper/misc/BaseController.kt

```diff
@@ -0,0 +1,14 @@
+package com.devindi.wallpaper.misc
+
+import android.arch.lifecycle.ViewModelStore
+import android.arch.lifecycle.ViewModelStoreOwner
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import com.bluelinelabs.conductor.archlifecycle.LifecycleController
+
+abstract class BaseController(args: Bundle? = null) : LifecycleController(args), ViewModelStoreOwner {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/5ebbb000266990b23827f5b4392520c161ebb501/app/src/main/java/com/devindi/wallpaper/misc/BaseController.kt#L9"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>